manifests/files.pp

```diff
@@ -1,4 +1,5 @@
 class postfix::files {
+  include postfix
```

Comment:
Actually, I don't think it makes sense to call that class publicly, so we might as well make it private and consider that `postfix` is instantiated:

```suggestion
  assert_private()
```

Reply:
Agree, fixed!